### ISSUE TITLE
longevity is not able to progress 

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla-cluster-tests#219]
commit 979a8df172f3938d63f0de7c430feabede2c25c0

extract from http://jenkins.cloudius-systems.com:8080/job/scylla-1.6-longevity/label=master/3/consoleFull

```
Cluster longevity-1-6-scylla-loader-set-d376668f (AMI: ami-63504874 Type: c3.large): Stress script content:
mkfifo /tmp/cs_pipe_$1_$2; cat /tmp/cs_pipe_$1_$2|python /usr/bin/cassandra_stress_exporter & cassandra-stress write cl=QUORUM duration=1800m -schema keyspace=keyspace$1 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -node 172.30.0.124|tee /tmp/cs_pipe_$1_$2; pkill -P $$ -f cassandra_stress_exporter; rm -f /tmp/cs_pipe_$1_$2
.
.
.
.
0
Remote [centos@54.166.115.61]: Running 'nodetool cfstats keyspace1'
[54.166.115.61] Running '/bin/ssh  -a -x  -o ControlPath=/var/tmp/ssh-master0WNg0b/socket -o StrictHostKeyChecking=no -o UserKnownHostsFile=/var/tmp/tmphsiQsR -o BatchMode=yes -o ConnectTimeout=300 -o ServerAliveInterval=300 -l centos -p 22 -i /var/tmp/longevity-1-6-sct-e0ed9db3.pem 54.166.115.61 "nodetool cfstats keyspace1"'
[54.144.210.214] [stdout] total,        145854,   23953,   23953,   23953,    41.8,    40.3,    73.8,    97.3,   122.7,   134.4,    7.2,  0.07374,      0,      0,       0,       0,       0,       0
[54.144.210.214] [stdout] total,        176286,   25032,   25032,   25032,    39.9,    36.3,    77.2,    92.0,   108.5,   121.7,    8.4,  0.06864,      0,      0,       0,       0,       0,       0
[54.166.115.61] [stdout] nodetool: Unknown keyspace: keyspace1
[54.166.115.61] [stdout] See 'nodetool help' or 'nodetool help <command>'.
```

it seems that the test to check the size of the keyspace is not aligning with the cassandra stress keyspace used.

**Comment by slivne** [comment]
moving back to 838cff4805df0eee47268b74780d1918c7277c63 seems to work (at least with regards to this)

http://jenkins.cloudius-systems.com:8080/job/scylla-1.6-longevity/label=master/4/console

**Comment by roydahan** [comment]
@kongove, please take a look on this one.
Seems like your support of multiple schema broke it.

**Comment by kongove** [comment]
This issue is fixed by #223
